content/cloud/develop.md

```diff
@@ -305,20 +307,22 @@ This command created a directory with the necessary files needed to build and ru
 <!-- @nocpy -->
 
 ```toml
-spin_manifest_version = "1"
-authors = ["Your Name <your-name@example.com>"]
+spin_manifest_version = 2
+
+[application]
+authors = ["tpmccallum <tim.mccallum@fermyon.com>"]
```

Comment:
Hmm, this actually switches it _back_.  Have you been the victim of a merge woe?